model/sifrovatko.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+use \Defuse\Crypto\Key,
+  \Defuse\Crypto\Crypto,
+  \Defuse\Crypto\Exception\CryptoException;
+
+class Sifrovatko
+{
+
+
+  /**
+   * @param $text
+   * @return string
+   * @throws \Defuse\Crypto\Exception\EnvironmentIsBrokenException
+   */
+  public function zasifruj($text): string
```

Comment:
Jelikož se nedělá DI, zasifruj i desifruj se s výhodou dají udělat statické, jak to ostatně šikovně navrhli i kluci v té Crypto knihovně.